.github/workflows/build-linux.yml

```diff
@@ -14,7 +14,7 @@ env:
 jobs:
   build:
 
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-18.04
```

Comment:
@medakk , does this not build on Ubuntu 16?

Reply:
OpenVDB needs atleast GCC 6.3.1. Ubuntu 16.04 comes with something older.

Reply:
Roger.  I thought we were requiring GCC 7 anyway.

I just set the older Ubuntu to make a binary that would be more likely to be compatible with older linuxes.

Reply:
Ah. We can't `apt-get install gcc-7` in Ubuntu 16.04 without adding a non-standard repo.